framework/src/executioners/TransientBase.C

```diff
@@ -208,18 +197,27 @@ TransientBase::TransientBase(const InputParameters & parameters)
     if (_num_steps == 0) // Always do one step in the first half
       _num_steps = 1;
   }
+
+  if (isParamValid("steady_state_convergence"))
+    _problem.setSteadyStateConvergenceName(getParam<ConvergenceName>("steady_state_convergence"));
+  else
+    _problem.setNeedToAddDefaultSteadyStateConvergence();
 }
 
 void
 TransientBase::init()
 {
   _problem.execute(EXEC_PRE_MULTIAPP_SETUP);
   _problem.initialSetup();
+  _fixed_point_solve->initialSetup();
 
   mooseAssert(getTimeStepper(), "No time stepper was set");
 
   _time_stepper->init();
 
+  auto & conv = _problem.getConvergence(_problem.getSteadyStateConvergenceName());
```

Comment:
so we always have one? even when we are not looking to detect steady state? 
I think this is in line with the prior code that always computed `_solution_change_norm` but I dont think we need to